packages/core/src/modules/dids/domain/key-type/keyDidJsonWebKey.ts

```diff
@@ -0,0 +1,20 @@
+import type { KeyDidMapping } from './keyDidMapping'
+import type { VerificationMethod } from '../verificationMethod'
+
+import { Key } from '../../../../crypto'
+import { AriesFrameworkError } from '../../../../error'
+import { getJsonWebKey2020VerificationMethod } from '../verificationMethod'
+import { VERIFICATION_METHOD_TYPE_JSON_WEB_KEY_2020, isJsonWebKey2020 } from '../verificationMethod/JsonWebKey2020'
+
+export const keyDidJsonWebKey: KeyDidMapping = {
```

Comment:
Not something for now, but I personally find these kinds of objects that contains various attributes and functions confusing. I know this is normal in the world of JS, but I don't think it's in line with the rest of the code base.

Reply:
In this particular case, I'm also not a fan of how it's implemented. But I do think it's quite a normal pattern to use, and don't see anything wrong with it. If you're only interested in static methods, an object that implements an interface is fine I think. 

Would you prefer to refactor this to a class?

Reply:
I agree with your point, but I just think it's very different from the rest of the code base. I would prefer refactoring to a class, purely out of consistency and readability reasons.

This is a personal preference, though. I'm not a fan of the JS way of things ;)

Reply:
Sure we can refactor to a class. Just need to think about better naming for this paradigm